changelogs/.changes.yaml

```diff
@@ -627,4 +627,5 @@ releases:
     - win_uri-junk-data.yaml
     - yum-handle-obsoletes-check-update.yaml
     - yumdnf-autoremove.yaml
+    - 49123-backport-devuan-support.yml
```

Comment:
This file is updated automatically during a release and should not be manually updated.

Reply:
so all I need is the file "49123-backport-devuan-support.yml" and nothing more?

Reply:
yes, remove reference from `changelogs/.changes.yaml` and leave `changelogs/fragments/49123-backport-devuan-support.yml` as is.

Reply:
done, thank  you for your patience